packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -518,13 +500,13 @@ def _get_dcos_diagnostics_health(z: zipfile.ZipFile, item: str):
             # namelist() gets a list of all items in a zip archive.
             logging.info(z.namelist())
 
-            # summaryErrorsReport.txt and summaryReport.txt are diagnostic job log files.
```

Comment:
Now the test for `summaryReport.txt` is missing, isn't it?

Reply:
Yes we discussed it. It does not bring any value and was removed from the new API